<launch>

  <!-- Front Left K79 Interface Node -->
  <node name="k79_front_left" pkg="radar_ros_interface" type="k79_node" output="screen" required="true" >
    <param name="host_ip" value="10.0.0.75" />
    <param name="host_port" value="1027" />
    <param name="radar_ip" value="10.0.0.13" />
    <param name="radar_port" value="7" />
    <param name="frame_id" value="front_left_radar_link" />
  </node>
  
  <!-- Front Right K79 Interface Node -->
  <node name="k79_front_right" pkg="radar_ros_interface" type="k79_node" output="screen" required="true" >
    <param name="host_ip" value="10.0.0.75" />
    <param name="host_port" value="1024" />
    <param name="radar_ip" value="10.0.0.10" />
    <param name="radar_port" value="7" />
    <param name="frame_id" value="front_right_radar_link" />
  </node>

</launch>
